app/app/controllers/webhooks/argyle/events_controller.rb

```diff
@@ -234,6 +234,7 @@ def log_sync_finish(payroll_account, report)
         paystubs_gross_pay_amounts_median: paystub_gross_pay_amounts[paystub_gross_pay_amounts.length / 2],
         paystubs_gross_pay_amounts_average: paystub_gross_pay_amounts.sum.to_f / paystub_gross_pay_amounts.length,
         paystubs_gross_pay_amounts_min: paystub_gross_pay_amounts.min,
+        paystubs_days_since_last_pay_date: report.paystubs.map { |p| Date.parse(p.pay_date) }.compact.max&.then { |last_pay_date| (Date.current - last_pay_date).to_i },
```

Comment:
Maybe the paystubs are already sorted by pay date? Might be worth verifying. Then maybe this could be simplified like:

```suggestion
        paystubs_days_since_last_pay_date: report.paystubs.last.then { |last_paystub| (Date.current - Date.parse(last_paystub.pay_date)).to_i if last_paystub },
```